src/neo/Trie/MPT/MPTNode/FullNode.cs

```diff
@@ -0,0 +1,51 @@
+using Neo.Cryptography;
+using Neo.IO;
+using System.IO;
+
+namespace Neo.Trie.MPT
+{
+    public class FullNode : MPTNode
+    {
+        public const int CHILD_COUNT = 17;
+        public MPTNode[] Children = new MPTNode[CHILD_COUNT];
+
+
+        public FullNode()
+        {
+            nType = NodeType.FullNode;
+            for (int i = 0; i < Children.Length; i++)
+            {
+                Children[i] = HashNode.EmptyNode();
+            }
+        }
+
+        protected override byte[] GenHash()
+        {
+            var bytes = new byte[0];
+            for (int i = 0; i < Children.Length; i++)
+            {
+                bytes = bytes.Concat(Children[i].GetHash());
+            }
+            return bytes.Sha256();
```

Comment:
I  think we should keep the same with `MerkleTree`, using `Crypto.Hash256` to replace `Sha256`. What do you think?@KickSeason

Reply:
ok